src/OptionT.ts

```diff
@@ -353,6 +405,17 @@ class Some<T> extends OptionT<T> {
   }
 
   unwrap(message?: string): T {
+    if (!this.hasBeenInspected) {
+      throw new Error(
+        'Called unwrap without first checking if it was safe to do so. Please verify' +
+          ' that the OptionT in question is a `Some` value before calling this function.'
```

Comment:
"... or use one of the other methods that provide safe defaults, like `unwrapOr`" - something like that - I think in general we should encourage people to not use `unwrap` where possible just because of this need to check first if it is safe.

Reply:
agree, I'll re-word here as well